setup.cfg

```diff
@@ -7,3 +7,6 @@ python-file-with-version = import_export/__init__.py
 
 [isort]
 profile = black
+
+[flake8]
+max-line-length = 110
```

Comment:
Could we add this in a separate PR?  I don't think it adds anything at present as we don't have flake8 integration, but it would be great to add in future.

Reply:
OK, I removed it. I have set up flake8 in VIM, so it provides me more managable amount of flake8 errors that I see. I will add new PR.

Reply:
@matthewhegarty I have added some linting checks to GitHub actions in #1369. I could try to make the code comply to those checks, but I would like to that after this MR is merged and only if you agree with the settings of those linters.